examples/doubkesSketchExample.go

```diff
@@ -0,0 +1,138 @@
+//+build mage
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/davecgh/go-spew/spew"
+	"github.com/grafadruid/go-druid"
+	"github.com/grafadruid/go-druid/builder"
+	"github.com/grafadruid/go-druid/builder/aggregation"
+	"github.com/grafadruid/go-druid/builder/datasource"
+	"github.com/grafadruid/go-druid/builder/dimension"
+	"github.com/grafadruid/go-druid/builder/granularity"
+	"github.com/grafadruid/go-druid/builder/intervals"
+	"github.com/grafadruid/go-druid/builder/postaggregation"
+	"github.com/grafadruid/go-druid/builder/query"
+	"log"
+	"time"
+)
+
+func getConnection() *druid.Client {
+	d, err := druid.NewClient("https://getafix-qal.odldruid-ppd.a.intuit.com")
+	if err != nil {
+		log.Fatal(err)
+	}
+	status, _, err := d.Common().Status()
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("{\"version\": \"" + status.Version + "\"}")
+	return d
+}
+
+/*
+	Prerequisite: These examples will work only if you have tdigest sketch data in your datastore.
+	To experiment, you can use the doubles_sketch_data.tsv file attached in this repo. It is a copy of  https://github.com/apache/druid/blob/master/extensions-contrib/tdigestsketch/src/test/resources/doubles_sketch_data.tsv
+*/
+
+// tdigestSketchUsingBuilder example using Builder Pattern
+func doublesSketchUsingBuilder() {
+	d := getConnection()
+	table := datasource.NewTable().SetName("o11y-gsapi-apigw")
```

Comment:
maybe change the table name for something more generic ? like `double-sketch` or so ?